.github/workflows/covector-comment-on-fork.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019-2024 Tauri Programme within The Commons Conservancy
```

Comment:
```suggestion
# Copyright 2019-2023 Tauri Programme within The Commons Conservancy
```

Reply:
the script hasn't been updated and expects 2023, let's update the script in another PR